### ISSUE TITLE
18F and FS review and ensure that all documentation is up to date

### ISSUE THREAD

**Issue by cmajel** [18F/NRM-Grants-Agreements#386]
**Background**
This relates to success criteria 15: _A team member wants to make a change to the current application and wonders what went into the initial decision. They find and access documentation from earlier phases of the project that informs their decision to make a different sort of change and they update the documentation accordingly (Box drive, GitHub wiki, etc.). _

The goal is for us to be updating documentation regularly. This is an issue to review our current wiki documentation and identify what, if any, adjustments are needed before the vendor team comes on board.

Timing of this depends on when a vendor comes on board, and may pair nicely with #385 

**We'll know we're done when...**
- [x] We've reviewed our documentation and edited, added, or removed documentation so that our documentation feels honest, clear and up to date.
- [x] Future team members have explanation and direction on where to find user stories and where to pick up them up.
- [x] Future team members know where the project stands

**Tasks**
- [ ] 
- [ ]

**Comment by cmajel** [comment]
Documenting this thought. As I was looking for past work on share of agreements by type, I thought that a short primer on grants and agreements and surface some key research artifacts that we found useful in addition to artifacts we've created.

A few things I'm thinking of
* Short write up on what's currently handled by the system (grants, agreements of various types) and what is not (contracting).
* [G&A User guide](https://docs.google.com/document/d/1ngFEuIN5rtF-zx3WiIxnp31KWTtBmOMV/edit#heading=h.30j0zll)
* Agreement decision tree
* [Distribution of agreements by type](https://drive.google.com/file/d/1uMOL5QDgTZemIYbd-kUsifIZwhn3aRQB/view?usp=sharing)
* Defining some key terms/systems (provision, grant vs agreement, FMMI, RACA, iWeb)

This also relates to #299
